docs/bot/configuration.md

```diff
@@ -22,6 +22,7 @@ These are the keys you can specify:
 | `imageSize`                      | Size (in px) of the user's avatar.                                                                  | Default: `100`                                                                                              |
 | `commit`                         | Auto-commit badge when adding contributors.                                                         | Default: `false`                                                                                           |
 | `commitConvention`               | Commit convention ([`angular`](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines), [`atom`](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages), [`ember`](https://guides.emberjs.com/v1.10.0/contributing/#toc_commits), [`eslint`](https://eslint.org/docs/1.0.0/developer-guide/contributing#step-2-make-your-changes), [`jshint`](https://jshint.com/contribute/), [`gitmoji`](https://gitmoji.carloscuesta.me/), or `none`).                                                         | Default: `angular`                                                                                           |
+| `commitType`                     | Set the `tag` for the commit in your chosen `commitConvention` (read more at [`commit-conv`](https://github.com/Berkmann18/commit-conv#parameters))    | Default: `docs`                                          |
```

Comment:
Is this link right? It doesn't look like the right repo, and includes little/no helpful information about which options can be used for the commitType option.

Would https://www.conventionalcommits.org be better?

Can anyone add information about whether the it is supported to add a `scope` to `commitType` as well?

e.g. will `"commitType": "chore(contributors)"` work?

Reply:
@tenshiAMD perhaps you know ☝🏻

Reply:
I see, that's the library that the all contributors bot uses, presumably. In that case, makes sense, though it's unclear how opinionated that library is about the commitType or if any string value would be accepted.